pkg/pods/pod.go

```diff
@@ -548,13 +548,13 @@ func (pod *Pod) getLaunchable(launchableStanza LaunchableStanza, runAsUser strin
 			Id:               launchableId,
 			RunAs:            runAsUser,
 			ConfigDir:        pod.EnvDir(),
-			Fetcher:          uri.DefaultFetcher,
 			RootDir:          launchableRootDir,
 			P2exec:           pod.P2exec,
 			RestartTimeout:   restartTimeout,
 			CgroupConfig:     launchableStanza.CgroupConfig,
 			CgroupConfigName: launchableStanza.LaunchableId,
 		}
+		ret.SetFetcher(uri.DefaultFetcher)
```

Comment:
I would have preferred renaming the field to having to make a method for this but couldn't think of a better name. Should this function be on the `launch.Launchable` interface if i keep it?